Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -506,24 +506,24 @@ void RenderFlexibleBox::paintChildren(PaintInfo& paintInfo, const LayoutPoint& p
     }
 }
 
-void RenderFlexibleBox::repositionLogicalHeightDependentFlexItems(Vector<LineState>& linesState, LayoutUnit gapBetweenLines)
+void RenderFlexibleBox::repositionLogicalHeightDependentFlexItems(FlexLineStates& lineStates, LayoutUnit gapBetweenLines)
```

Comment:
This naming looks redundant to me. We only have flex lines in flex layout.

Reply:
though it's spec language so probably fine (not sure how much the type should reflect it as oppose to the variable name. in FFC we do it the other way around, like LinesCrossSizeList flexLinesCrossSizeList, guess it's a matter of taste)